packages/workbox-build/src/lib/utils/warn-about-config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const logHelper = require('../log-helper');
+
+/**
+ * A helper to find and log any configuration options that were set which are
+ * not valid for a given usage mode.
+ *
+ * @param {Array<String>} blacklist The names of invalid configuration options.
+ * @param {Object<String,String>} fullConfig The configuration options to check.
+ */
+module.exports = (blacklist, fullConfig) => {
+  const invalidConfig = Object.keys(fullConfig)
+    .filter((configKey) => blacklist.indexOf(configKey) !== -1);
```

Comment:
Could you use includes?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
We're still aiming for Node 4 compatibility with these modules, so no `Array.prototype.includes()`: http://node.green/#ES2016-features-Array-prototype-includes